docs/getting_started/hopsworksai/guides/getting_started.rst

```diff
@@ -233,12 +232,11 @@ You are also able to stop or terminate the instance.
 Step 3: Outside Access to the Feature Store
 -------------------------------------------
 
-By default, only the Hopsworks UI is exposed (made available to clients on external networks, like the Internet)
-by your Hopsworks instance. To integrate with external platforms and
-access APIs for services such as the Feature Store, you have to expose them.
+By default, only the Hopsworks UI is made available to clients on external networks, like the Internet.
+To integrate with external platforms and access APIs for services such as the Feature Store, you have to open ports.
```

Comment:
`you have to open ports` sound strange to me, I would say `to open the services ports`

Reply:
done